database/changesets/20210422_TerraWorkspaceSinkTable.xml

```diff
@@ -15,14 +15,27 @@
         <comment>
             the 'Terra Workspace' sink table schema
         </comment>
-        <sql>
-            CREATE TYPE TerraWorkspaceSink AS (
-                id              bigint,      -- unique to this table
-                workspace       text,        -- the fully-qualified 'Namespace/Name' of the Terra workspace to execute workflows in
-                entity          text,        -- the entity type to import data from the source as
-                fromOutputs     text         -- mapping of how to coerce the output of executor to sink
-            )
-        </sql>
+        <createTable tableName="TerraWorkspaceSink">
+            <column name="id" type="bigint" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="workspace" type="text"
+                    remarks="the fully-qualified 'Namespace/Name' of the Terra workspace to execute workflows in">
+                <constraints nullable="false"/>
+            </column>
+            <column name="entity" type="text"
+                    remarks="the entity type to import data from the source as">
+                <constraints nullable="false"/>
+            </column>
+            <column name="from_outputs" type="text"
+                    remarks="mapping of how to coerce the output of source to executor in serialized EDN">
```

Comment:
I think this should be "coerce the output of executor to sink".

Reply:
Nice catch!!

Reply:
👍

Reply:
I suppose in this case, it's coercing to an entity in the workspace